Would appear `MINSIGSTKSZ` is not constant on linux:
```
  # .---command stderr------------
  # | /home/gha/actions-runner/_work/llvm-project/llvm-project/compiler-rt/test/asan/TestCases/Posix/multiple_sigaltstack.cpp:8:6: error: variable length array declaration not allowed at file scope
  # |     8 | char global_alt_stack[MINSIGSTKSZ];
  # |       |      ^                ~~~~~~~~~~~
  # | 1 error generated.
  # `-----------------------------
  # error: command failed with exit status: 1
```

--- a/compiler-rt/test/asan/TestCases/Posix/multiple_sigaltstack.cpp
+++ b/compiler-rt/test/asan/TestCases/Posix/multiple_sigaltstack.cpp
@@ -5,7 +5,11 @@
 #include <stdio.h>
 #include <stdlib.h>
 
+#if (__APPLE__)
+char global_alt_stack[MINSIGSTKSZ];
+#else
 char global_alt_stack[4096 * 4];
+#endif
 
 int main() {
   stack_t altstack;

